src/components/Tag/Tag.stories.mdx

```diff
@@ -0,0 +1,223 @@
+import { Meta, Preview, Props, Story } from '@storybook/addon-docs';
+import { boolean, select, text, withKnobs } from '@storybook/addon-knobs';
+import Tag from './Tag';
+import Typography from '../Typography';
+import Stack from '../storyUtils/Stack';
+import { FIGMA_URL, Function } from '../../utils/common';
+import SectionHeader from '../../storybook/SectionHeader';
+import TagShowcase from '../../storybook/Showcases/TagShowcase';
+import { getIconSelectorKnob } from '../../utils/stories';
+import { useState } from 'react';
+
+<Meta
+  title="Design System/Tag"
+  component={Tag}
+  parameters={{
+    design: [
+      {
+        type: 'figma',
+        name: 'Tag',
+        url: `${FIGMA_URL}?node-id=3325%3A58246`,
+      },
+    ],
+    chromatic: { diffThreshold: 0.3 },
+  }}
+/>
+
+<SectionHeader title={'Tag'} />
+
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+- [Variants](#variants)
+
+## Overview
+
+A universal Tag that is used to highlight system-generated metadata.
+
+## Props
+
+<Props of={Tag} />
+
+## Usage
+
+<UsageGuidelines
+  guidelines={[
+    'Use tags to show metadata that needs to be highlighted somehow',
+    'Use tags to relate secondary information about a page element (px a “Writer” tag next to a name)s',
+    'Use selectable tags to quickly filter existing content',
+    'Do not use tags as standalone elements in a page; they should always relate to something elseh',
```

Comment:
What i would expect for `do not use` cases is to have another section of being defined in another array here and display it accordingly. This applies to all sections that we have

For example
```
<UsageGuidelines guidelines={[]} policies={[]} //do not use cases />
```